tutorials/README.md

```diff
@@ -4,16 +4,23 @@ We currently provide two tutorials to get you started with Snorkel.
 
 ### Intro Tutorial
 
-In this tutorial, we go over the basics of Snorkel by 
-building an application to identify mentions of spouses in a corpus 
+In this tutorial, we go over the basics of Snorkel by
+building an application to identify mentions of spouses in a corpus
 of news articles. We keep this one well manicured.
 
-
 ### CDR Tutorial
 
 For a more advanced tutorial, check out the CDR tutorial, where
 we build an application to extract chemical-disease relations
 from academic literature. We check that it runs end-to-end using
 automatic testing, but it's also under active development so
-you may run into small issues along the way. So it goes with 
+you may run into small issues along the way. So it goes with
 real-world information extraction!
+
+### `Fonduer` Tutorial
+
+For an advanced tutorial for extracting information from _richly formatted
+data_, check out the `Fonduer` tutorials. In the `hardware` tutorial, we extract
```

Comment:
Can you add an "--e.g. tables, PDFs, etc.--"?  Since I don't think the term "richly formatted data" will mean much to many people?